frontend/src/components/widgets/DataGrid/index.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
```

Comment:
nit: 2022 (i wonder how this happened as the others are correct 🤔)

Reply:
oh, good catch. No clue why this is 2021 😄